data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2163,6 +2163,11 @@ spec:
                       for the cluster. If set, the AMI should belong to the same region
                       as the cluster.
                     type: string
+                  bestEffortDeleteIgnition:
```

Comment:
Nit: Can we add the default behavior here? Sample text below, feel free to tweak it.

`The default value to `false` means that the errors encountered during deletion of Ignition objects from their S3 bucket would not be ignored. Enable this functionality when there are known reasons disallowing deletion of S3 objects (and you expect the deletion of S3 objects to encounter errors.) `

I understand that the field being deprecated didn't have all of this. This is an opportunity to improve it.

Reply:
Done